python/tk_framework_desktopserver/shotgun/api_v2.py

```diff
@@ -1300,51 +1310,56 @@ def _legacy_process_configs(self, config_data, entity_type, project_id, all_acti
                     entity_type,
                     config_path
                 )
+
+                all_actions[config_name] = dict(
+                    actions=[],
+                    config=config_entity,
+                )
+
                 continue
 
             if raw_actions_data["retcode"] != 0:
                 logger.error(
                     "A shotgun_get_actions call did not succeed: %s",
```

Comment:
What does this error message look like for support? Should we include the yml file path or the config descriptor to make it more helpful?